src/crypto/transactionCache.js

```diff
@@ -168,7 +234,7 @@ export class TransactionCache {
 
   // Returns number of updated transactions
   _checkUpdatedTransactions(transactions: Array<Transaction>): number {
-    // Logger.debug('_updateTransactions', transactions)
+    Logger.debug('_updateTransactions', transactions)
```

Comment:
presumably this should be `_checkUpdatedTransactions` instead of `_updateTransactions` in the log statement